pkg/cluster/install.go

```diff
@@ -192,3 +194,14 @@ func (i *manager) addResourceProviderVersion(ctx context.Context) error {
 	})
 	return err
 }
+
+// addArchitectureVersion sets architecture version in
+// the cluster document for version-tracking purposes.
+func (i *manager) addArchitectureVersion(ctx context.Context) error {
+	var err error
+	i.doc, err = i.db.PatchWithLease(ctx, i.doc.Key, func(doc *api.OpenShiftClusterDocument) error {
+		doc.OpenShiftCluster.Properties.ArchitectureVersion = version.ArchitectureVersion
```

Comment:
This to match version.GitCommit so we keep all version-specific info in one place.

Reply:
could go in the 4.5 PR?